src/components/code/index.tsx

```diff
@@ -23,17 +23,17 @@ function scrollToBottom(elementRef: Element): void {
   setTimeout(() => elementRef?.scrollTo?.(0, elementRef.scrollHeight), 0);
 }
 
-export const Code = ({
+export const Code: FunctionComponent<CodeProps & { children?: string }> = ({
```

Comment:
children was set manually (`CodeProps & { children?: string }`) to avoid having the Code component allowing non-string values to be used.

The Code component is used to show logs and similar in text/string form and the typing in `PropsWithChildren` will not reflect that, as it will allow many other types to be passed as the children object as well

![image](https://github.com/equinor/radix-web-console/assets/32248439/e156f1c3-8741-4320-bde5-2b94eec63f67)
![image](https://github.com/equinor/radix-web-console/assets/32248439/b310a7df-7d30-476f-a36c-002ca6720e0f)